content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -208,7 +209,30 @@ http://localhost:9090/graph
 
 For more details, see [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics/#about-the-prometheus-add-on).
 
-### ServiceGraph
+### Kiali and ServiceGraph
```

Comment:
You need two headings here. One level three heading that encompasses both Kiali and ServiceGraph. The second heading should then be Kiali. I suggest:
```suggestion
### Install mesh visualization tools

#### Kiali
```